mir-module/src/main/resources/xsl/import/datacite2mods.xsl

```diff
@@ -39,7 +42,7 @@
             <mods:role>
                 <mods:roleTerm authority="marcrelator" type="code">aut</mods:roleTerm>
             </mods:role>
-            <xsl:apply-templates select="nameIdentifier[@nameIdentifierScheme='ORCID']" />
+            <xsl:apply-templates select="nameIdentifier[string-length(mcrxml:getMatchingString(text(), $orcidRegex)) = (19)]" />
```

Comment:
Not sure if it is wise to ignore the `nameIdentifierScheme`-attribute. Could lead to false positives. 

I'd relax this part to `<xsl:apply-templates select="nameIdentifier" />` (or leave it as is) ...